Makefile

```diff
@@ -405,7 +405,10 @@ meson-config meson-cfg meson-conf:
 	echo TODO
 
 meson: build
+	@echo [ SDB Build ]
+	python sys/meson_sdb.py
```

Comment:
better use $(PYTHON) and define PYTHON?=python on top, this way the user can override it